maro/simulator/scenarios/oncall_routing/business_engine.py

```diff
@@ -250,11 +257,23 @@ def step(self, tick: int) -> None:
 
         # Interrupt and throw decision event
         if (tick + 1) % self._config["interrupt_cycle"] == 0 and len(self._oncall_order_buffer) > 0:
+            route_meta_info_dict = {}
+            for route in self._routes:
+                t = self._current_dest_arrival_tick[route.name]
+                route_meta_info_dict[route.name] = {
+                    "carrier_idx": route.carrier_idx,
+                    "current_destination_arrival_duration": None if t is None else t - tick  # TODO
+                }
+
             decision_event = self._event_buffer.gen_decision_event(
                 tick=tick,
                 payload=OncallRoutingPayload(
                     get_oncall_orders_func=lambda: list(self._oncall_order_buffer.values()),
-                    get_routes_info_func=lambda: self._routes,
+                    get_route_plan_dict_func=lambda: {
+                        _route.name: [_elem.order for _elem in _route.remaining_plan] for _route in self._routes
+                    },
+                    get_estimated_duration_predictor_func=lambda: self.estimated_duration_predictor,
```

Comment:
why not return self._estimated_duration_predictor in the lambda function directly?